resources/config.rb

```diff
@@ -0,0 +1,82 @@
+#
+# Cookbook Name:: grafana
+# Resource:: config
+#
+# Copyright 2014, Jonathan Tron
+# Copyright 2017, Andrei Skopenko
+# Copyright 2018, Sous Chefs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Configures the installed grafana instance
+
+property  :instance_name,       String, name_property: true
+property  :env_directory,       String, default: '/etc/default'
+property  :owner,               String, default: 'grafana'
+property  :group,               String, default: 'grafana'
+property  :restart_on_upgrade,  String, default: 'false'
+property  :conf_directory,      String, default: '/etc/grafana'
+property  :config_file,         String, default: lazy { ::File.join(conf_directory, 'grafana.ini') }
+property  :app_mode,            String, default: 'production', equal_to: %w(production development)
+property  :cookbook,            String, default: 'grafana'
+property  :source,              String, default: 'grafana.ini.erb'
+
+action :install do
+  user new_resource.owner do
```

Comment:
any resource 


without config in between the do end blocks can get rid of the do end blocks. 
e.g. 
```suggestion
  user new_resource.owner
```